lib/enums/query/props/prop_enum.py

```diff
@@ -1,6 +1,10 @@
+from __future__ import annotations
```

Comment:
Are you sure this works on Python 3.8? 
I believe I opted not to use this because of version dependencies when using python 3.6

Reply:
We've dropped 3.6 with the upgrade, and I've just checked 3.7+ https://docs.python.org/3/library/__future__.html